crates/humanode-rpc/src/lib.rs

```diff
@@ -136,15 +149,18 @@ pub struct Deps<C, P, BE, VKE, VSF, A: ChainApi, SC> {
 }
 
 /// Instantiate all RPC extensions.
-pub fn create<C, P, BE, VKE, VSF, A, SC>(
+pub fn create<C, P, BE, VKE, VSF, A, SC, EC>(
     deps: Deps<C, P, BE, VKE, VSF, A, SC>,
 ) -> Result<RpcModule<()>, Box<dyn std::error::Error + Send + Sync>>
 where
     BE: Backend<Block> + 'static,
     BE::State: StateBackend<sp_runtime::traits::HashFor<Block>>,
     C: ProvideRuntimeApi<Block> + BlockBackend<Block> + StorageProvider<Block, BE> + AuxStore,
     C: BlockchainEvents<Block>,
-    C: HeaderBackend<Block> + HeaderMetadata<Block, Error = BlockChainError> + 'static,
+    C: HeaderBackend<Block>
+        + CallApiAt<Block>
```

Comment:
Doesn't `ProvideRuntimeApi` imply `CallApiAt`?

Reply:
Let's group this `CallApiAt` trait bound together with `ProvideRuntimeApi` one above on a separate new line of bounds

Reply:
Looks like it doesn't as I had an error here. Let me check.

Just checked, it doesn't - https://github.com/humanode-network/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/primitives/api/src/lib.rs#L697

https://github.com/humanode-network/substrate/blob/033d4e86cc7eff0066cd376b9375f815761d653c/primitives/api/src/lib.rs#L646

Reply:
Yep, must've changed recently and we missed it, no big deal

Reply:
https://github.com/humanode-network/humanode/pull/829/commits/49cc501b3e4cb5b787d2b2b56ce0e6fe155ba09c